dragonfly/session/handler_player_action.go

```diff
@@ -46,13 +46,17 @@ func handlePlayerAction(action int32, face int32, pos protocol.BlockPos, entityR
 		}
 	case protocol.PlayerActionStopSwimming:
 		s.c.StopSwimming()
+	case protocol.PlayerActionContinueDestroyBlock:
+		fallthrough
```

Comment:
I think this is supposed to above the protocol.PlayerActionCrackBreak case, or is it sent as a sort of 'start break' action?

Reply:
Yeah it's used to start breaking the next block as far as I can tell, hence why I fall through there

Reply:
Got it, that makes sense.